purchase_order_import/tests/test_order_response_import.py

```diff
@@ -555,29 +567,26 @@ def test_15(self):
         self.assertEqual(self.purchase_order.state, "purchase")
         self.assertEqual(len(self.purchase_order.picking_ids), 2)
         move_ids = self.line1.move_ids
-        self.assertEqual(len(move_ids), 3)
+        self.assertEqual(len(move_ids), 2)
         self.assertEqual(sum(move_ids.mapped("product_qty")), self.line1.product_qty)
         move_confirmed = move_ids.filtered(
-            lambda s: s.state == "assigned" and s.product_qty == confirmed_qty
+            lambda s: s.state == "assigned" and s.product_qty == 2
         )
         self.assertTrue(move_confirmed)
         self.assertEqual(
             _("%s items should be delivered into a next delivery.") % "2",
             move_confirmed.note,
         )
         move_cancel = move_ids.filtered(
-            lambda s: s.state == "cancel" and s.product_qty == 1
+            lambda s: s.state == "cancel" and s.product_qty == 8
```

Comment:
I'm a bit worried that the cancelled qty is now the original quantity minus the backorder quantity. While the total delivered and backorder quantities are correct, this should not be necessary given that recent version of Odoo allow to adapt the quantities of confirmed stock moves. Do you think that we can use that to end up with a cancelled quantity of 1 again (which seems the best outcome for an original quantity of 10 minus 7 delivered and 2 backordered).